### ISSUE TITLE
Comment-`var`-comment-variable - both comments glued

### ISSUE THREAD

**Issue by Fr0sT-Brutal** [pasdoc/pasdoc#44]
Check following code
```
// configurable globals
var
  // sample var
  Foo: byte;
```
It makes `Foo` have description `configurable globals sample var`. If we try to `@exclude` the first comment, the variable gets excluded too.

**Comment by michaliskambi** [comment]
That''s because of " Glueing single-line comments" described on https://github.com/pasdoc/pasdoc/wiki/WhereToPlaceComments . For PasDoc, this is equivalent to

```
var
  // configurable globals
  // sample var
  Foo: byte;
```

We could probably fix it by stopping the  Glueing single-line comments" feature when it encounters some token between comments.

**Comment by Fr0sT-Brutal** [comment]
Yes, I guess breaking glueing on a token appearance would be logical. Meanwhile it could be solved with `{}` comment:
```
{ configurable globals }
var
  // sample var
  Foo: byte;
```
In the future I'm thinking about configurable comment-ignore-markers so that f.ex. comments starting with `~` would be totally skipped. What do you think?

**Comment by michaliskambi** [comment]
I looked into breaking of "Glueing single-line comments". It doesn't seem trivial looking at PasDoc_Parser code. It's an old code, I would refactor many things there now...

For now I'll leave this task.

**Comment by Fr0sT-Brutal** [comment]
OK, no problem, the issue is less actual now with `--ignore-marker`.
